### ISSUE TITLE
.NET Standard / .NET 5 / .NET 6 support?

### ISSUE THREAD

**Issue by DurtyFree** [Willy-Kimura/BetterFolderBrowser#6]
Is it possible to add .NET 5 / 6 or .NET Standard support?

Not sure if its technically possible, but it would be nice to see those supported.

**Comment by Willy-Kimura** [comment]
Yes there is. I'll be sure to work on supporting both .NET 5 and 6.
